package.json

```diff
@@ -23,21 +23,23 @@
     "index.cjs"
   ],
   "dependencies": {
+    "execa": "^8.0.1",
     "postcss-styled-syntax": "0.5.0",
     "stylelint-config-css-modules": "4.3.0",
-    "stylelint-config-recommended": "13.0.0",
-    "stylelint-config-recommended-scss": "13.1.0"
+    "stylelint-config-recommended": "14.0.0",
+    "stylelint-config-recommended-scss": "14.0.0"
   },
   "devDependencies": {
     "@types/prettier": "2.7.3",
     "eslint": "8.55.0",
     "eslint-config-upleveled": "7.3.4",
-    "stylelint": "15.11.0",
+    "stylelint": "16.0.2",
     "stylelint-config-upleveled": "1.0.3",
```

Comment:
can we remove this? it was only used with the config for the tests, right? and now we config using `.`

Reply:
removed 
[8ded186](https://github.com/upleveled/stylelint-config-upleveled/pull/56/commits/8ded186f314a117e201de4a0320510f3e0c021cf)